container-structure-test.yaml

```diff
@@ -37,5 +37,6 @@ commandTests:
         value: Hello There
     command: "entrypoint"
     expectedOutput:
```

Comment:
this seems wrong to me - why would we wan't the actual command printed out?

Reply:
Just to make sure we are on the same page, when running jbang-action it echoes the actual command to be executed in the first line (`jbang hello@jbangdev A B C`):
```
$ docker run -it --rm jbangdev/jbang-action hello@jbangdev A B C
jbang hello@jbangdev A B C
[jbang] Building jar...
Hello A
Hello B
Hello C
```
Are you saying that we might have not needed to print this line?  I personally find it useful when setting up the action, especially for debugging to make sure I'm passing correct args. It might be good idea to introduce a new input option like `printcmd` and disable the cmd printing by default, but it's another story.

Reply:
Yeah - it is actually a mistake it gets echoed by default; especially on stdout. 

Printing the command if option set like printcmd as you suggest would be more appropriate fix. Even better if you make it go to stderr so the docker command will work as a jbang replacement.